sdk/keyvault/keyvault-keys/CHANGELOG.md

```diff
@@ -1,11 +1,11 @@
 # Release History
 
-
 ## 4.2.0-beta.3 (Unreleased)
 
 - [Breaking] Removed `dist-browser` from the published package. To bundle the Azure SDK libraries for the browsers, please read our bundling guide: [link](https://github.com/Azure/azure-sdk-for-js/blob/master/documentation/Bundling.md).
 - Updated the Key Vault Keys Long Running Operation Pollers to follow a more compact and meaningful approach moving forward.
 - Bug fix: The logging of HTTP requests wasn't properly working - now it has been fixed and tests have been written that verify the fix.
+- Consolidate LocalCryptographyClient and KeyVaultCryptographyClient into a single client that can be run in either mode and add KeyId to the public API of CryptographyClient.
```

Comment:
Do you mind mentioning something about being able to receive a key ID, or a KeyVaultKey, or a JsonWebKey? Could be meaningful for the user.